src/components/App.js

```diff
@@ -1,16 +1,63 @@
 import React from 'react';
-
+import Menu from './Menu';
 import '../styles/App.scss';
 
 class App extends React.Component {
   constructor(){
     super();
+
+    this.state = {menuItems:[]};
+    this.fetch = this.fetch.bind(this);
+    this.receiveOrder = this.receiveOrder.bind(this);
+    this.postOrder = this.postOrder.bind(this);
   }
 
+componentDidMount(){
+  this.fetch()
+}
+
+receiveOrder(){
+  this.setState();
+  postOrder(this.state.order)
+}
+
+postOrder(order){
+
+  fetch('http://localhost:8080/api/order', {
+    method: 'post',
+    body: JSON.stringify(order),
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  }).then(function(response) {
+    return response.json();
+  }).then(data => {
+    console.log(data)
+    // handle response
+  });
+
+
+}
+
+fetch(){
```

Comment:
I would give this method a more descriptive name. Fetch is a little too ambiguous